defaults/main.yml

```diff
@@ -31,3 +33,17 @@ couchdb_pki_cert_suffix: '.pem'
 
 couchdb_init: systemd
 couchdb_activate: True
+couchdb_async_io_threads: 16
```

Comment:
i wonder if we shouldn't set a default here (so let couch use its default) and just let it be overridden by the implementer